tests/MongoDB.Driver.Tests/ConnectionsSurvivePrimaryStepDownTests.cs

```diff
@@ -99,7 +108,7 @@ public void Connection_pool_should_not_be_cleared_when_replSetStepDown_and_GetMo
                     RunOnSecondary(client, secondary.EndPoint, BsonDocument.Parse("{ replSetFreeze : 0 }"));
                 }
 
-                var replSetStepDownCommand = BsonDocument.Parse("{ replSetStepDown : 20, force : true }");
+                var replSetStepDownCommand = BsonDocument.Parse("{ replSetStepDown : 30, force : true }");
```

Comment:
Why was it 20? Why is it 30 now?

Does the spec say to use 30?

Reply:
yep, it's the spec requirement: https://github.com/mongodb/specifications/blob/master/source/connections-survive-step-down/tests/README.rst#getmore-iteration
As far as I understand, there is no significant difference between these values, but the spec requires it now, so I've changed it